smdebug/mxnet/utils.py

```diff
@@ -42,22 +42,3 @@ def get_reduction_of_data(aggregation_name, tensor_data, tensor_name, abs=False)
         op = f(tensor_data)
         return op
     raise RuntimeError("Invalid aggregation_name {0} for mx.NDArray".format(aggregation_name))
-
-
-def make_numpy_array(x):
-    if isinstance(x, np.ndarray):
-        return x
-    elif np.isscalar(x):
-        return np.array([x])
-    elif isinstance(x, NDArray):
-        return x.asnumpy()
-    elif isinstance(x, tuple):
-        # todo: fix this, will crash
```

Comment:
what is this about?

Reply:
The crash used to occur because of a ref to dtype.
I have removed it and validated the support of tuple in `tests/core/test_make_numpy_array.py` line 26